pkg/cli/cmd/bicep/publish/publish.go

```diff
@@ -246,6 +246,10 @@ func (r *Runner) prepareDestination(ctx context.Context) (*remote.Repository, er
 		Credential: ds.Get,
 	}
 
+	if strings.HasPrefix(r.Destination.host, "localhost:") || strings.HasPrefix(r.Destination.host, "127.0.0.1:") {
```

Comment:
Is this going to be the case all the time for localhost? Can I serve localhost on HTTPS? Should this be like an input from the user and if it is empty and localhost then we can set it to HTTP?

We can also have a separate function called **isLocalhost** to split the localhost checking logic.

Reply:
yes, its going to be the case with localhost all the time and it will not be able to serve localhost with https. e.g: https://github.com/sajayantony/obom/blob/main/pkg/oras.go#L85